WearBuildConfig/mobile/build.gradle

```diff
@@ -0,0 +1,71 @@
+apply plugin: 'com.android.application'
+apply from: "$rootProject.projectDir/WearBuildConfig/shared-config/android-signing.gradle"
+
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.0"
+
+    defaultConfig {
+        applicationId "com.novoda.demo.wearbuildconfig"
+        minSdkVersion 16
+        targetSdkVersion 25
+        versionCode 1
+        versionName "1.0"
+
+        buildConfigField 'boolean', 'PLAY_SERVICES', "true"
```

Comment:
Thanks for the hint, but this field is just an example of a value set differently in the two variations of the flavour dimension.
I'd prefer to keep it as simple as possible, without adding external plugins in this demo